.github/workflows/build-and-deploy.yml

```
@@ -2,49 +2,152 @@ name: Build & Deploy
+            ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}/server
+            ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}/client
+          tags: |
+            type=ref,event=branch
+            type=sha,prefix={{branch}}-
+
+      - name: Build and push image (tagged as both server and client)
+        uses: docker/build-push-action@471d1dc4e07e5cdedd4c2171150001c434f0b7a4 # v6.15.0
+        with:
+          context: .
+          file: ./Dockerfile
+          file: ./server/Dockerfile
```

actionlint (1.7.10)

[error] key "file" is duplicated in "with" section. previously defined at line:143,col:11. note that this key is case insensitive

(syntax-check)